ClosedXML_Tests/Excel/Sparklines/SparklinesTests.cs

```diff
@@ -0,0 +1,1075 @@
+﻿using ClosedXML.Excel;
```

Comment:
Do a CodeMaid organize on all *new* files and add the `// Keep ...` header, please.

Reply:
This class is insanely thorough!